.gitignore

```diff
@@ -1,3 +0,0 @@
-.idea
-/config/config.local.php
-phpunit.xml
```

Comment:
You should no remove this file already included within the repo. Thanks.

Reply:
added again.